pkgs/tools/typesetting/tex/texlive/build-texlive-package.nix

```diff
@@ -118,17 +118,20 @@ let
       {
         passthru = commonPassthru // { tlType = "bin"; };
         inherit meta;
-        # shebang interpreters
-        buildInputs = (args.extraBuildInputs or [ ]) ++ [ bash perl ] ++ (lib.attrVals (args.scriptExts or [ ]) extToInput);
+        # shebang interpreters and compiled binaries
+        buildInputs = let outName = builtins.replaceStrings [ "-" ] [ "_" ] pname; in
```

Comment:
this doesn't work for `biber-ms`.

Reply:
so `outName` should only be used for `core.${outName}` and `core-big.${outName}`, of course, done